src/notify-button/notify-button.html

```diff
@@ -52,14 +69,32 @@
             if (name != null && name !== "") {
                 if (!this._isBlocked()) {
                     var date = new Date();
-                    var ajax = this.shadowRoot.querySelector('#ajax')
-                    ajax.body = { name, date };
-                    ajax.generateRequest();
+                    var firebaseRequest = this.shadowRoot.querySelector('#firebase')
+                    firebaseRequest.body = {name, date};
+                    firebaseRequest.generateRequest();
+
+                    var content = name + " möchte dich gerne treffen.\n Dies war am: " + date;
+
+                    var appToken = this.person.pushapptoken;
+                    var apiToken = this.person.pushapitoken;
+                    var deviceID = this.person.pushdevice;
+
+                    console.log(this.person);
+                    console.log(apiToken);
```

Comment:
Pls don't log this

Reply:
Der Akku war auf 3% und es musste schnell gehen

Reply:
Kein Problem :stuck_out_tongue_closed_eyes: